workflow/applications/gfs_cycled.py

```diff
@@ -230,13 +231,14 @@ def get_task_names(self):
 
         if self.do_gempak:
             gfs_tasks += ['gempak']
+            gfs_tasks += ['gempakmeta']
+            gfs_tasks += ['gempakncdcupapgif']
+            gfs_tasks += ['gempakpgrb2spec']
+            gfs_tasks += ['npoess_pgrb2_0p5deg']
```

Comment:
```suggestion
            gfs_tasks += ['gempak', 'gempakmeta', 'gempakncdcupagif', 'npoess_pgrb2_0p5deg', 'gempakpgrb2spec']
```
gempakpgrb2spec depends on npoess_pgrb2_0p5deg so it should come first.  order matters in the xml